db/migrate/20230310082546_change_category_setting_num_auto_bump_daily_default.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ChangeCategorySettingNumAutoBumpDailyDefault < ActiveRecord::Migration[7.0]
+  def up
+    change_column_default :category_settings, :num_auto_bump_daily, 0
+
+    execute(<<~SQL)
+      UPDATE category_settings
+      SET num_auto_bump_daily = 0
+      WHERE num_auto_bump_daily IS NULL;
```

Comment:
This check is here because we have populated values from custom fields in a previous PR.

Reply:
Hmm, the values of `num_auto_bump_daily` have been populated earlier? Has the existing code been writing to the custom field and the new `category_settings` table? Or is there a chance that we will lose values that have only been stored in the custom field since `category_settings` was populated?

Reply:
You are correct! Some time has passed since then, and we need to re-sync to catch any updates that have gone into custom fields since then. Script incoming. 🙂

Reply:
@gschlager I finally had a chance to revisit this. I added more data migration code to the migration, as well as a migration spec. Please have a look. 🙏